portfolio/src/main/webapp/script.js

```diff
@@ -16,13 +16,69 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+    const greetings =
+        ['My favorite food is donuts.',
+            'I\'ve had surgery 3 times.',
+            'My favorite TV show of all time is Prison Break.',
+            'I play basketball.'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+    // Pick a random greeting.
+    const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+
+    // Add it to the page.
+    const greetingContainer = document.getElementById('greeting-container');
+    greetingContainer.innerText = greeting;
+
+
+}
+function mod1PopUp() {
```

Comment:
might be good to put some short documentation outlining the purpose of these functions.